kubernetes-metrics-server.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  metrics_server_chart_name = "metrics-server"
```

Comment:
Same here - I don't see the value of the locals here.

Reply:
se above